lib/std/os/linux.zig

```diff
@@ -2001,6 +2001,58 @@ pub fn fremovexattr(fd: usize, name: [*:0]const u8) usize {
     return syscall2(.fremovexattr, fd, @intFromPtr(name));
 }
 
+pub const sched_param = extern struct {
+    sched_priority: i32,
+};
+
+pub const SCHED = packed struct(i32) {
+    pub const Mode = enum(u3) {
+        /// normal multi-user scheduling
+        OTHER = 0,
```

Comment:
```suggestion
        NORMAL = 0,
```

To match UAPI headers.